apps/demos/Demos/Chat/Overview/Angular/app/app.component.ts

```diff
@@ -0,0 +1,62 @@
+import { NgModule, Component, enableProdMode } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { Observable } from 'rxjs';
+
+import notify from 'devextreme/ui/notify';
+
+import { DxChatModule } from 'devextreme-angular';
+import { User, Message } from 'devextreme/ui/chat';
+import { ChatService } from './chat.service';
```

Comment:
For what purpose are we renaming app.service to chat.service? All demos use the same convention for naming app.service.ts.

Reply:
78922d32e8627e3e28cd47962cb322fa1486d7b1